pkg/storage/ec/client.go

```diff
@@ -139,22 +139,9 @@ func (ec *ecClient) Get(ctx context.Context, nodes []*proto.Node, es eestream.Er
 				ch <- rangerInfo{i: i, rr: nil, err: err}
 				return
 			}
-			ps, err := ec.d.dial(ctx, n)
-			if err != nil {
-				zap.S().Errorf("Failed getting piece %s -> %s from node %s: %v",
-					pieceID, derivedPieceID, n.GetId(), err)
-				ch <- rangerInfo{i: i, rr: nil, err: err}
-				return
-			}
-			rr, err := ps.Get(ctx, derivedPieceID, pieceSize, &pb.PayerBandwidthAllocation{})
-			// no ps.CloseConn() here, the connection will be closed by
-			// the caller using RangeCloser.Close
-			if err != nil {
-				zap.S().Errorf("Failed getting piece %s -> %s from node %s: %v",
-					pieceID, derivedPieceID, n.GetId(), err)
-				ch <- rangerInfo{i: i, rr: nil, err: err}
-				return
-			}
+
+			rr := &lazyPieceRanger{dialer: ec.d, n: n, id: derivedPieceID, size: pieceSize, pba: &pb.PayerBandwidthAllocation{}}
```

Comment:
It would be more readable if you break this initialization into multiple lines:
```go
rr := &lazyPieceRanger{
	dialer: ec.d,
	n:      n,
	id:     derivedPieceID,
	size:   pieceSize,
	pba:    &pb.PayerBandwidthAllocation{},
}
```